apps/pie-docs/src/_includes/layouts/page-layout.njk

```diff
@@ -9,5 +11,5 @@ layout: layouts/default-layout.njk
     {% include "content-page.njk" %}
 {% else %}
     {# By default we will show the coming soon graphic for an empty page #}
```

Comment:
May need to update this now, as it looks like you've removed the coming soon as the default?